src/components/SEO.tsx

```diff
@@ -39,22 +40,23 @@ const SEO: React.FC<{
       titleTemplate={title ? `%s / ${site.siteMetadata.title}` : undefined}
       meta={[
         {
-          name: `description`,
-          content: metaDescription
+          name: `application-name`,
+          content: site.siteMetadata.title
         },
         {
-          property: `og:title`,
-          content: title
-        },
-        {
-          property: `og:description`,
+          name: `description`,
           content: metaDescription
         },
         {
-          property: `og:type`,
-          content: `website`
+          name: `msapplication-config`,
+          content: `browserconfig.xml`
         }
       ].concat(meta!)}
+      link={[
+        { rel: `mask-icon`, href: 'mask-icon.svg', color: PALETTE.kale[700] },
+        { rel: `apple-touch-icon`, href: 'apple-touch-icon.png' },
+        { rel: `shortcut icon`, href: 'favicon.ico' }
```

Comment:
I'm curious why these strings are backticked?

Reply:
Opps, I should have backticked `favicon.ico` and `apple-touch-icon.png` to keep consistent with what was originally in this file. 

I think Gatsby [defaults to using backticks](https://github.com/gatsbyjs/gatsby/blob/master/starters/default/src/components/seo.js#L39) in their starter project. I was going to call this out in an earlier PR by Austin, but then I noticed that it seems to be Gatsby's default. 

I don't see a reason to use string template literals if there are no embedded expressions. But since it was a Gatsby default, I didn't intend to change it.

Reply:
I would contend the Garden practice supersedes Gatsby. We don't use backticks when they aren't needed.

Reply:
The file has been updated and removes unnecessary string template usages.